all-the-stars-js/readme.md

```diff
@@ -38,82 +37,67 @@ files are located there:
-2. From the [Google API console](https://code.google.com/apis/console)
-  `API Access` section, create a client ID for a chrome app. See
-  [Chrome Apps Identity](https://developer.chrome.com/apps/app_identity.html)
-  for how to obtain a stable app ID by adding a `key` to `manifest.json`.
-3. Edit the `manifest.json` file and add your client ID from the API console.
+written using HTML/JS/CSS.  To begin, you will need to create your own version
+of the game in the Play Console. Once you have done that, you will create a
+Chrome Webstore entry for your application and place the resulting client IDs
+and keys into your `manifest.json` file. To follow this process, perform the
+following steps:
+
+1. Follow the
+   [packaging instructions](https://developer.chrome.com/extensions/packaging#creating)
+   to locally package your copy of all-the-stars.  You will end up with a crx
+   file and a key file.  Note the instructions on the latter half of the page
+   for uploading the crx to the Web Store; you'll need those later when you want
+   to publish your game.
+2. Load the crx into chrome.  You can do this by going to the
+   chrome://extensions page and dragging the crx that was created in Step 1 into
+   Chrome.
+3. Find the new entry on the extensions page for your app, and note the ID.  It
+   will be a long string of letters, like "cfhblmkcgnkbkbobakpeikejjgjjlfhi".
+4. Create your application in the Play Console, as described in our [Developer
+   Documentation](https://developers.google.com/games/services/console/enabling).
+   If you have already created this application in the Play Console (because you
+   have created and run the Android or iOS version of the gamne, for example), you
+   can re-use your current version of the game.
+5. Link your chrome app to your new PGS entry.
+    * On the "Linked apps" section, click "Chrome" to link a chrome app.
+    * Under "Chrome Application ID", enter the app ID from step 3.
+6. Authorize your app and get a client ID.
+    * Click "Authorize Your App" on the next screen.  Confirm that the
+      app ID is correct.
+    * You should see a message noting that your Client ID was successfully
+      linked.  Note the Client ID.  You'll need it for the next step.
+7. Replace the client ID in the application.
+    * In the `manifest.json` file, replace the `oauth2.client_id` with your
+      Client ID from step 6.
+8. Follow the instructions on the
+   [packaging instructions](https://developer.chrome.com/extensions/packaging#creating)
+   page to update your app, now that you have added the client ID.
+9. Load the crx into chrome.
 
 That's it! Your application should be ready to run!
 
 ## Troubleshooting
 
-**I see a some kind of `400 bad request` error in my JavaScript console when
-calling `GET https://accounts.google.com/o/oauth2/auth...`. What does that
-mean?**
-
-This could be due to a number of different causes, but probably the most
```

Comment:
They do, the new errors and remedial action to take are described in the second bullet point, the one that starts of saying "If you see oauth failures in the javascript console of your app..."